client/next.config.ts

```
@@ -3,13 +3,14 @@ import createNextIntlPlugin from "next-intl/plugin";
 
 const withNextIntl = createNextIntlPlugin({
   experimental: {
+    extract: true,
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)